assets/lokomotive-kubernetes/docs/flatcar-linux/google-cloud.md

```diff
@@ -18,7 +18,7 @@ Install [Terraform](https://www.terraform.io/downloads.html) v0.12.x on your sys
 
 ```sh
 $ terraform version
-Terraform v0.12.17
```

Comment:
We don't need to update those docs anymore. They should be removed.

Reply:
I was ambivalent about that, but then I thought maybe we can keep them for a while and follow a deprecation policy for those docs.

Reply:
What do you think @iaguis ?

Reply:
I think we should remove them, we'll make lokomotive-kubernetes read-only soon.